src/acl/external/AD_group/ext_ad_group_acl.cc

```diff
@@ -181,9 +181,9 @@ Get_primaryGroup(IADs * pUser)
     VariantInit(&var);
 
     /* Get the primaryGroupID property */
-    hr = pUser->lpVtbl->Get(pUser, L"primaryGroupID", &var);
+    hr = pUser->Get(BSTR(L"primaryGroupID"), &var);
```

Comment:
Just for my own edification, what error do you get if this explicit BSTR conversion is removed? Please ignore this question if it takes too long to answer.

```suggestion
    hr = pUser->Get(L"primaryGroupID", &var);
```

This optional question is not requesting any PR changes.

Reply:
```
../../../../../src/acl/external/AD_group/ext_ad_group_acl.cc: In function 'HRESULT Recursive_Memberof(IADs*)':
../../../../../src/acl/external/AD_group/ext_ad_group_acl.cc:429:20: error: ISO C++ forbids converting a string constant to 'BSTR' {aka 'wchar_t*'} [-Werror=write-strings]
  429 |     hr = pObj->Get(L"memberOf", &var);
      |                    ^~~~~~~~~~~
```

Reply:
Thank you! Using that error, I found an [article](https://pvs-studio.com/en/docs/warnings/v745/) that, AFAICT, claims that the changes proposed by this PR are buggy: "This code builds (compiles and links) correctly, but it will not function properly because the string does not have a length prefix."

I still do not know enough to be _certain_ that this PR is buggy, but I believe this concern has to be addressed, so I have to block this PR.